vignettes/mmrm_review_methods.Rmd

```diff
@@ -0,0 +1,104 @@
+## Data Sets
+
+The FEV data set is contains measurements of FEV1 (forced expired volume in one second), a measure of how quickly the lungs can be emptied. Low levels of FEV1 may indicate chronic obstructive pulmonary disease (COPD). This data set is used to illustrate model fitting with the `mmrm`, `lme4`, `nlme`, `glmmtmb` R packages as well as `SAS`.
+
+The FEV data set
```

Comment:
@chstock @BrianLang, from your experience, are there any practically relevant aspects missing that the FEV data does not cover sufficiently, do we need more example data?

Reply:
I think it is good dataset for the basic explanations. One feature that I think might be of particular interest is when the number of subjects is rather small (<30 or 40).